src/wintoastlib.h

```diff
@@ -142,6 +145,7 @@ namespace WinToastLib {
         std::wstring                        _imagePath{};
         std::wstring                        _audioPath{};
         std::wstring                        _attributionText{};
+        std::wstring                        _scenario = L"Default";
```

Comment:
Following the other members, lets use aggregate initialization

Reply:
Done